src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -21,7 +21,8 @@ public static Monument convertCsvRowToMonument(String csvRow) {
         Monument result = new Monument();
 
         for (int columnIndex = 0; columnIndex < csvRowArray.length; columnIndex++) {
-            String value = csvRowArray[columnIndex];
+            // Grab the value at the current column and remove any quotes that may be left
+            String value = csvRowArray[columnIndex].replace("\"", "");
```

Comment:
what if the title is actually supposed to have quotes in it? this should probably only check for quotes at the beginning & end

Reply:
Yup, working on that right now